packages/project-input-validator/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# @sap-ux/project-input-validator
+
+## 0.1.0
+
+### Minor Changes
+
+-   9967c5f: Initial release of reusable modules for validating the input constraints required by Fiori projects. 
+This library provides a reusable validation utilities to support validations required in tools that support
+Fiori project generation, deployment configration, etc. The initial set of validation functions supports 
+validation used in Fiori project deployment configurations.
```

Comment:
I would have expected this file to be mostly empty and the changeset populates this text instead.

Reply:
Removed manually added changelog